src/react-components/input/ToolbarButton.scss

```diff
@@ -42,6 +42,14 @@
   }
 }
 
+:local(.small) {
+  width: 48px;
+}
+
+:local(.small) :local(.icon-container) {
+  flex-shrink: 0;
+}
+
```

Comment:
All this `small` css stuff is just to make the text wrap under the toolbar icons rather than grow their width now that I've given the two chat panels longer labels to differentiate them